nncf/common/scopes.py

```diff
@@ -70,7 +70,7 @@ def should_consider_scope(
     :return: A boolean value specifying whether a serializable_id should be considered (i.e. "not ignored", "targeted")
     """
     string_id = str(serializable_id)
-    return (target_scopes is None or matches_any(string_id, target_scopes)) \
+    return (not target_scopes or matches_any(string_id, target_scopes)) \
```

Comment:
?

Reply:
because `target_scopes` can be an empty List

Reply:
If `target_scopes` is an empty list, I would expect that this should mean that there are no target scopes, e.g. everything in the model is _not_ targeted.